### PR TITLE
Zjiang10/word frequency

### PR DESCRIPTION
**Pull request by zongyi001**
[withTest] refactor this project

### CODE REVIEW

**Comment by WZD09** [comment]
good job!

**Comment by xYYux** [comment]
niceeeeeeeeee job

**Comment by 5eeya** [comment]
good job~

File: WordFrequency/WordFrequencyGame.cs [hunk 1]
```diff
@@ -1,75 +1,55 @@
...
-                {
-                    Input input = new Input(entry.Key, entry.Value.Count);
-                    list.Add(input);
-                }
+            return string.Join("\n", wordFrequencyList.ToArray());
 
-                inputList = list;
+        }
+
+        private List<string> getWordListMap(ref List<WordFrequency> inputList)
```
Comment (kgaohahaha) [inline WordFrequency/WordFrequencyGame.cs]: useless function

**Comment by zongyi001** [comment]
refactor

**Pull request closed without merging**